.github/workflows/e2e-tests.yml

```diff
@@ -27,25 +29,31 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.AWS_REGION }}
-      - name: Show current linux user
-        run: whoami
-      # - name: AWS CLI test
-      #   run: aws configure list
-      - name: Checkout
+      - name: Git Checkout
         uses: actions/checkout@v2
-      # Install NPM dependencies, cache them correctly
-      # and run all Cypress tests
-      - name: Install dependencies and build
+      - name: Get NPM Cache Dir
+        id: npm-cache
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
+      - name: Enable NPM Cache on package-lock file

```

Comment:
Just curious, can you clarify what's going on here?

Reply:
Just getting nmp config get output to be used later.

See https://github.com/actions/cache/blob/master/README.md

And
https://github.com/actions/cache/blob/master/examples.md#using-multiple-systems-and-npm-config